NetKAN/RealPlume-StockConfigs.netkan

```diff
@@ -0,0 +1,32 @@
+{
+    "spec_version"      : "v1.4",
+    "$kref"             : "#/ckan/github/KSP-RO/RealPlume-StockConfigs",
+    "ksp_version"       : "1.0",
+    "name"              : "Real Plume - Stock Configs",
+    "identifier"        : "RealPlume-StockConfigs",
+    "abstract"          : "A set of effects and basic configs that can be attached to any engine as appropriate.  Requires secondary configs to function correctly.",
+    "license"           : "CC-BY-NC-SA",
+    "depends" : [
+        { "name" : "ModuleManager" },
+        { "name" : "SmokeScreen" },
```

Comment:
You can remove depends on SmokeScreen and ModuleManager. It will inherit the dependencies from RealPlume. The configs don't actually depend on those mods. If you change dependencies for RealPlume you'll only have to change them in that netkan instead of both.